.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Do not convert line endings as is directly compared with in spec
+spec/fixtures/saved-html.html -text
```

Comment:
atom/atom's .gitattributes does this a bit differently: https://github.com/atom/atom/blob/master/.gitattributes#L1